esp-link/cgiwifi.c

```diff
@@ -238,6 +241,16 @@ static int ICACHE_FLASH_ATTR cgiWiFiStartScan(HttpdConnData *connData) {
   return HTTPD_CGI_DONE;
 }
 
+// Start scanning, API interface
```

Comment:
Can you consolidate the code a bit and have one function call the other? Flash isn't infinite...